src/Command/AbstractClientRequestCommand.php

```diff
@@ -68,6 +71,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ->setRaw($input->getOption('raw') !== false)
             ->setDefaultAuthMethod($this->defaultAuthMethod);
 
+        // In case of a dry-run the request to TER is skipped.
+        if ($input->getOption('dry-run') !== false) {
```

Comment:
I think `!== false` is superfluous here, right?

Reply:
That's a stan maxlevel requirement but should be !== true, same mistake as below.